cypress/support/stackable/util.js

```diff
@@ -60,11 +60,18 @@ export function changeUnit( unit = '', name = '', isInPopover = false ) {
 	if ( unit ) {
 		selector()
 			.then( $baseControl => {
-				if ( $baseControl.find( '.ugb-base-control-multi-label__units', { log: false } ).length ) {
-					selector()
-						.find( 'button', { log: false } )
-						.contains( containsRegExp( unit ) )
-						.click( { force: true, log: false } )
+				if ( Cypress.env( 'STACKABLE_VERSION' ) === 2 ) {
+					if ( $baseControl.find( '.ugb-base-control-multi-label__units', { log: false } ).length ) {
+						selector()
+							.find( 'button', { log: false } )
+							.contains( containsRegExp( unit ) )
+							.click( { force: true, log: false } )
+					}
+				} else if ( Cypress.env( 'STACKABLE_VERSION' ) === 3 ) {
+					if ( $baseControl.find( '.stk-control-label__units' ).length ) {
+						selector().find( '.stk-control-label__units button.is-active' ).click( { force: true } )
+						selector().find( `.stk-control-label__units button[data-value="${ unit }"]` ).click( { force: true } )
+					}
```

Comment:
Don't use `else if` here.

Prematurely end the function after the first `if` statement instead. With this, the command will still run even if `STACKABLE_VERSION` is not defined.